src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Retrieves updated column header order and generates a header string.
+ *
+ * Example: Changes this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section|Team|Name|Email|Comments\n"
+ *
+ * @param handsontableColHeader
```

Comment:
Applies for all: if you don't have any other information about the parameter, best to just omit it.